### PR TITLE
added b4 table

### PR DESCRIPTION
**Pull request by Sefadzi**

### CODE REVIEW

File: scripts/ento/entoDashboard/DESCRIPTION [hunk 1]
```diff
@@ -34,10 +34,13 @@ Imports:
     shinyWidgets,
     shinyjs,
     dashboardthemes,
-    here
+    here,
```
Comment (joebrew) [inline scripts/ento/entoDashboard/DESCRIPTION:37]: `dashboardthemes` is a package used, and should be called here

File: scripts/ento/entoDashboard/NAMESPACE [hunk 2]
```diff
@@ -8,4 +8,9 @@ importFrom(golem,add_resource_path)
 importFrom(golem,bundle_resources)
 importFrom(golem,favicon)
 importFrom(golem,with_golem_options)
+importFrom(htmltools,HTML)
+importFrom(htmltools,tagAppendAttributes)
+importFrom(htmltools,tagList)
+importFrom(htmltools,tags)
+importFrom(shiny,column)
```
Comment (joebrew) [inline scripts/ento/entoDashboard/NAMESPACE:15]: `dashboardthemes` is used in the app and needs to be in the NAMESPACE

**Review by joebrew — COMMENTED** [review]
@Sefadzi Please epxand the README.me to include exactly what a developer needs to do to run this. For example, it appears that the app expects certain environment variables, but it is not clear to another developer that those environment variables should be exported prior to running the app.

**Comment by joebrew** [comment]
When I run `Rscript run_dev.R` from within the `dev` directory, I get an error, @Sefadzi, even after exporting the environment variables needed to run and after logging in:

```
Error: Missing argument for option: p

Launch the GUI with: java -jar /home/joebrew/Documents/briefcase/ODK-Briefcase-v1.18.0.jar
Launch a CLI operation with: java -jar /home/joebrew/Documents/briefcase/ODK-Briefcase-v1.18.0.jar <operation> <params>

Available operations:
  -c,--clear_prefs                     Clear saved preferences
  -e,--export                          Export a form
  -h,--help                            Show help
  -pc,--pull_collect                   Pull from Collect
  -plla,--pull_aggregate               Pull form from an Aggregate instance
  -pllc,--pull_central                 Pull form from a Central server
  -psha,--push_aggregate               Push form to an Aggregate instance
  -pshc,--push_central                 Push form to a Central server
  -v,--version                         Show version

Params for -plla operation:
  -U,--odk_url <arg>                   ODK Server URL
  -p,--odk_password <arg>              ODK Password
  -sd,--storage_directory <arg>        Briefcase storage directory
  -u,--odk_username <arg>              ODK Username
Optional params for -plla operation:
  -id,--form_id <arg>                  Form ID
  -ii,--include_incomplete             Include incomplete submissions
  -mhc,--max_http_connections <arg>    Maximum simultaneous HTTP connections (defaults to 8)
  -sfd,--start_from_date <arg>         Start pull from date
  -sfl,--start_from_last               Start pull from last submission pulled

Params for -pshc operation:
  -E,--odk_email <arg>                 ODK Email
  -U,--odk_url <arg>                   ODK Server URL
  -p,--odk_password <arg>              ODK Password
  -pid,--project_id <arg>              ODK Project ID number
  -sd,--storage_directory <arg>        Briefcase storage directory
Optional params for -pshc operation:
  -id,--form_id <arg>                  Form ID
  -mhc,--max_http_connections <arg>    Maximum simultaneous HTTP connections (defaults to 8)

Params for -e operation:
  -ed,--export_directory <arg>         Export directory
  -f,--export_filename <arg>           Filename for export operation
  -id,--form_id <arg>                  Form ID
  -sd,--storage_directory <arg>        Briefcase storage directory
Optional params for -e operation:
  -em,--exclude_media_export           Exclude media in export
  -end,--export_end_date <arg>         Export end date (inclusive) (yyyy-MM-dd or yyyy/MM/dd)
  -ig,--include_geojson                Include a GeoJSON file with spatial data
  -oc,--overwrite_csv_export           Overwrite files during export
  -pb,--pull_before                    Pull before export
  -pf,--pem_file <arg>                 PEM file for form decryption
  -rgn,--remove_group_names            Remove group names from column names
  -sa,--smart_append                   Include only new submissions since last export
  -ssm,--split_select_multiples        Split select multiple fields
  -start,--export_start_date <arg>     Export start date (inclusive) (yyyy-MM-dd or yyyy/MM/dd)

Params for -pc operation:
  -od,--odk_directory <arg>            ODK directory
  -sd,--storage_directory <arg>        Briefcase storage directory
Optional params for -pc operation:
  -id,--form_id <arg>                  Form ID

Params for -psha operation:
  -U,--odk_url <arg>                   ODK Server URL
  -p,--odk_password <arg>              ODK Password
  -sd,--storage_directory <arg>        Briefcase storage directory
  -u,--odk_username <arg>              ODK Username
Optional params for -psha operation:
  -fsb,--force_send_blank              Force sending the blank form to the Aggregate instance
  -id,--form_id <arg>                  Form ID
  -mhc,--max_http_connections <arg>    Maximum simultaneous HTTP connections (defaults to 8)

Params for -pllc operation:
  -E,--odk_email <arg>                 ODK Email
  -U,--odk_url <arg>                   ODK Server URL
  -p,--odk_password <arg>              ODK Password
  -pid,--project_id <arg>              ODK Project ID number
  -sd,--storage_directory <arg>        Briefcase storage directory
Optional params for -pllc operation:
  -id,--form_id <arg>                  Form ID
  -mhc,--max_http_connections <arg>    Maximum simultaneous HTTP connections (defaults to 8)

The directory /tmp/RtmpU54XEU/ODK Briefcase Storage doesn't exist. Creating it
Error: Form entoa3 not found
Warning: Error in : '/tmp/RtmpU54XEU/entoa3.csv' does not exist.
  57: stop
  56: check_path
  55: standardise_path
  54: read_delimited
  53: readr::read_csv
  50: getEntoData [/home/joebrew/Documents/bohemia/scripts/ento/entoDashboard/R/fct_getEntoData.R#49]
  49: server [/home/joebrew/Documents/bohemia/scripts/ento/entoDashboard/R/app_server.R#30]
Error : '/tmp/RtmpU54XEU/entoa3.csv' does not exist.

```

**Comment by joebrew** [comment]
@Sefadzi After setting my environment variables and then logging in as admin/admin, I get an error:
```
── Column specification ──────────────────────────────────────────────────────────────────────────────
cols(
  .default = col_character()
)
ℹ Use `spec()` for the full column specifications.

Warning: Error in matrix: length of 'dimnames' [2] not equal to array extent
  54: matrix
  53: Ops.data.frame
  51: a4_ind_mosq [/home/joebrew/Documents/bohemia/scripts/ento/entoDashboard/R/fct_getEntoData.R#61]
  50: getEntoData [/home/joebrew/Documents/bohemia/scripts/ento/entoDashboard/R/fct_getEntoData.R#99]
  49: server [/home/joebrew/Documents/bohemia/scripts/ento/entoDashboard/R/app_server.R#30]
Error in matrix(if (is.null(value)) logical() else value, nrow = nr, dimnames = list(rn,  : 
  length of 'dimnames' [2] not equal to array extent

```

File: scripts/ento/entoDashboard/NAMESPACE (hunk 2 above)
Comment (Sefadzi) [reply scripts/ento/entoDashboard/NAMESPACE:15]: I am using shiny golem and the namespace file generated is supposed to be read-only.  dashboardthemes is already added/ imported in the DESCRIPTION file.

**Pull request merged**